pkg/pgmodel/ha/HA.go

```diff
@@ -0,0 +1,73 @@
+package ha
+
+import (
+	"sync"
+	"time"
+
+	"github.com/timescale/promscale/pkg/pgxconn"
+)
+
+type HA struct {
+	Cluster         string
+	Leader          string
+	LeaseStart      time.Time
+	LeaseUntil      time.Time
+	MaxTimeSeen     time.Time //max data time seen by any instance
+	MaxTimeInstance string    // the instance name that’s seen the maxtime
+	DBClient        pgxconn.PgxConn
+	LeaseTimeout    string
+	LeaseRefresh    string
```

Comment:
The `leaseTimeOut` & `leaseRefresh` are exposed as flags so I made the exportable fields. During Promscale startup the custom values can be assigned. 

I know we thought of having them as mutable in DB level. I will update accordingly.

Reply:
even if configurable by flags, since they are not supposed to change at runtime, it's better to have them set at init (through a constructor func, together with the passed PgxConn instance) and be unexported